<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TujianWindow</class>
 <widget class="QMainWindow" name="TujianWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>648</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>930</width>
    <height>626</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>930</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>图鉴模式</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="searchLine">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>500</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>搜索...</string>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::VisualMoveStyle</enum>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSearch">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background:transparent;}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Dishes.qrc">
      <normaloff>:/images/search.svg</normaloff>:/images/search.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTag">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background:transparent;border:none;
border-radius: 10px;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Dishes.qrc">
      <normaloff>:/images/tag-one.svg</normaloff>:/images/tag-one.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAdd">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background:transparent;border:none;
border-radius: 10px;}QPushButton:hover{background:rgb(213, 215, 215);}</string>
    </property>
    <property name="text">
     <string>添加菜品</string>
    </property>
    <property name="icon">
     <iconset resource="Dishes.qrc">
      <normaloff>:/images/add.svg</normaloff>:/images/add.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>861</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLeft">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>590</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background:transparent;border:none;
border-radius: 10px;}QPushButton:hover{background:rgb(213, 215, 215);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Dishes.qrc">
      <normaloff>:/images/left.svg</normaloff>:/images/left.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRight">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>590</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background:transparent;border:none;
border-radius: 10px;}QPushButton:hover{background:rgb(213, 215, 215);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Dishes.qrc">
      <normaloff>:/images/right.svg</normaloff>:/images/right.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="lbPage">
    <property name="geometry">
     <rect>
      <x>417</x>
      <y>590</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="SearchTagsArea">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none</string>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>491</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnNameSearch">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background:transparent;border:none;
border-radius: 10px;}QPushButton:hover{background:rgb(213, 215, 215);}</string>
    </property>
    <property name="text">
     <string>菜名检索</string>
    </property>
    <property name="icon">
     <iconset resource="Dishes.qrc">
      <normaloff>:/images/bowl-one.svg</normaloff>:/images/bowl-one.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTagSearch">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background:transparent;border:none;
border-radius: 10px;}QPushButton:hover{background:rgb(213, 215, 215);}</string>
    </property>
    <property name="text">
     <string>标签检索</string>
    </property>
    <property name="icon">
     <iconset resource="Dishes.qrc">
      <normaloff>:/images/tag.svg</normaloff>:/images/tag.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHome">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background:transparent;border:none;
border-radius: 10px;}QPushButton:hover{background:rgb(213, 215, 215);}</string>
    </property>
    <property name="text">
     <string>返回首页</string>
    </property>
    <property name="icon">
     <iconset resource="Dishes.qrc">
      <normaloff>:/images/home-two.svg</normaloff>:/images/home-two.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <zorder>searchLine</zorder>
   <zorder>btnSearch</zorder>
   <zorder>btnAdd</zorder>
   <zorder>stackedWidget</zorder>
   <zorder>btnLeft</zorder>
   <zorder>btnRight</zorder>
   <zorder>lbPage</zorder>
   <zorder>SearchTagsArea</zorder>
   <zorder>btnTag</zorder>
   <zorder>btnNameSearch</zorder>
   <zorder>btnTagSearch</zorder>
   <zorder>btnHome</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>930</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Dishes.qrc"/>
 </resources>
 <connections/>
</ui>
